File: my-client-angular/src/app/joonja/comment/comment.ts
export class Comment {
//   `c_no`             INT                         NOT NULL    AUTO_INCREMENT COMMENT '게시글번호',
// `c_writer`         VARCHAR(20)                 NOT NULL                   COMMENT '작성자',
// `c_writer_email`   VARCHAR(200)                NOT NULL                   COMMENT '작성자 이메일',
// `c_content`        VARCHAR(4000)               NOT NULL                   COMMENT '내용',
// `c_date`           DATE          DEFAULT now() NOT NULL                   COMMENT '작성일시',

  c_no : number;
  c_writer : string;
  c_writer_email : string;
  c_content : string;
  c_date : string;

}

File: my-server-spring/src/main/java/com/example/t3/comment/model/Comment.java
package com.example.t3.comment.model;


import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Comment {
	  private int c_no;
	  private int d_pageno;
	  private String c_writer;
	  private String c_pw;
	  private String c_content;
	  private Date c_date;
	

	
}
